### PR TITLE
#220 changed fontResourceId access modifier

### PR DESCRIPTION
**Pull request by anton6tak**
Added an android-app-compose sample, it doesn't work now

### CODE REVIEW

File: sample/android-app-compose/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+plugins {
+    id("android-app-convention")
+    id("dev.icerock.mobile.multiplatform-resources")
+    kotlin(module = "android")
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:4]: already enabled in `android-app-convention`

File: sample/android-app-compose/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+dependencies {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        google()
+    }
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:12]: why this here? remove it

File: sample/android-app-compose/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    implementation(dependencyNotation = "dev.icerock.moko:resources:0.16.2")
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:13]: use project accessor

File: sample/android-app-compose/build.gradle.kts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    implementation(dependencyNotation = "androidx.appcompat:appcompat:1.4.0-alpha03")
+    implementation(dependencyNotation = "androidx.compose.ui:ui:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.compose.ui:ui-tooling:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.compose.ui:ui-tooling-preview:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.compose.runtime:runtime:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.compose.material:material:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.compose.foundation:foundation:1.0.0-rc02")
+    implementation(dependencyNotation = "androidx.activity:activity-compose:1.3.0-rc02")
+    implementation("androidx.constraintlayout:constraintlayout:2.0.4")
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:22]: move libs to libs.toml and use here

File: sample/android-app-compose/build.gradle.kts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+android {
+    compileSdk = 31
+
+    defaultConfig {
+        minSdk = 21
+        targetSdk = 31
+        versionCode = 1
+        versionName = "1.0"
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:32]: already enabled by convention

File: sample/android-app-compose/build.gradle.kts [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+        applicationId = "dev.arturmavl.mrfjci"
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:33]: sample id as in other sample app here

File: sample/android-app-compose/build.gradle.kts [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
+        freeCompilerArgs = freeCompilerArgs +
+                "-Xopt-in=kotlin.RequiresOptIn" +
+                "-Xexplicit-api=strict"
+    }
+
+    composeOptions {
+        kotlinCompilerVersion = "1.5.10"
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:48]: but we have 1.5.20 no?

File: sample/android-app-compose/src/main/AndroidManifest.xml [hunk 8]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="dev.arturmavl.mrfjci.android">
```
Comment (Alex009) [inline sample/android-app-compose/src/main/AndroidManifest.xml:3]: sample package please

File: sample/android-app-compose/build.gradle.kts [hunk 9]
```diff
@@ -0,0 +1,51 @@
+plugins {
```
Comment (Alex009) [inline sample/android-app-compose/build.gradle.kts:1]: copyright please

File: sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/MainActivity.kt [hunk 10]
```diff
@@ -0,0 +1,36 @@
+package dev.arturmavl.mrfjci.android
```
Comment (Alex009) [inline sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/MainActivity.kt:1]: copyright please
and sample package

File: sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/MainActivity.kt [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+internal class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+    }
+}
+
+private val monserratFontFamily: FontFamily = FontFamily(
+    fonts = listOf(
+        Font(
+            resId = MontserratRegular.fontResourceId,
```
Comment (Alex009) [inline sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/MainActivity.kt:23]: `resId = MR.fonts.Montserrat.regular.fontResourceId`

File: sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/Test.kt [hunk 12]
```diff
@@ -0,0 +1,6 @@
+package dev.arturmavl.mrfjci.android
```
Comment (Alex009) [inline sample/android-app-compose/src/main/java/dev/arturmavl/mrfjci/android/Test.kt:1]: not needed file

File: sample/android-app-compose/src/main/res/layout/activity_main.xml [hunk 13]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Alex009) [inline sample/android-app-compose/src/main/res/layout/activity_main.xml:1]: not needed file at all

**Review by Alex009 — CHANGES_REQUESTED** [review]

File: sample/android-app-compose/src/main/java/com/icerockdev/MainActivity.kt [hunk 14]
```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Tetraquark) [inline sample/android-app-compose/src/main/java/com/icerockdev/MainActivity.kt:3]: Add empty line here after the copyright comment, please

File: sample/android-app-compose/build.gradle.kts [hunk 15]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Tetraquark) [inline sample/android-app-compose/build.gradle.kts:3]: Add empty line here after the copyright comment, please

**Review by Tetraquark — CHANGES_REQUESTED** [review]

**Review by Tetraquark — APPROVED** [review]

**Pull request merged**